CollisionDetection/collide.py

```diff
@@ -9,19 +10,26 @@
 from move import move_all
 
 
-# This ignores geometries we have said we don't care about
-# As there are only [(len(geometries)-1)!] combinations, and we don't care about some, there isn't much effort saved
-# by using spaces (which do a quicker estimate of collisions first)
-def collide(geometries, ignore):
+def collide(geometries, ignore, collision_func=ode.collide):
+    """
+    Calculates which of the given geometries will collide, ignoring geometries that are specified as ignored.
+    As there are only [(len(geometries)-1)!] combinations, and we don't care about some, there isn't much effort saved
+    by using spaces (which do a quicker estimate of collisions first)
```

Comment:
what is meant by using spaces?

Reply:
I'm not sure. I'd left the comment as it was, but as it is confusing I will remove it.